source/services/policyManager/lib/exports.ts

```diff
@@ -4,8 +4,11 @@
 /**
  *
  */
+import { UserAgent, UserAgentPair } from "@aws-sdk/types";
+
+const userAgentPair : UserAgentPair = [`${process.env.USER_AGENT_PREFIX}/${process.env.SOLUTION_ID}`, `${process.env.SOLUTION_VERSION}`]
+export const customUserAgent : UserAgent = [userAgentPair];
```

Comment:
@gsingh04 the useragent prefix is only parit of the user agent string built as a userAgentPair.

Reply:
do we have these env variables configured on the lambda resource?

Reply:
yes